specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json

```diff
@@ -0,0 +1,11996 @@
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "in": "query",
+            "name": "maxLastModifiedDateTime",
+            "description": "Maximum last modified date of resource (inclusive).",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "in": "query",
+            "name": "$maxPageSize",
+            "description": "Maximum number of items needed (inclusive).\r\nMinimum = 10, Maximum = 1000, Default value = 50.",
+            "type": "integer",
+            "format": "int32",
+            "default": 50,
+            "maximum": 1000,
+            "minimum": 10
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
+            "description": "Skip token for getting next set of results.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The requested API version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ApplicationDataListResponse"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ApplicationData_List": {
+            "$ref": "./examples/ApplicationData_List.json"
+          }
+        }
+      }
+    },
+    "/farmers/{farmerId}/application-data/{applicationDataId}": {
+      "get": {
+        "tags": [
+          "ApplicationData"
+        ],
+        "description": "Get a specified application data resource under a particular farmer.",
+        "operationId": "ApplicationData_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "farmerId",
+            "description": "ID of the associated farmer resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "applicationDataId",
+            "description": "ID of the application data resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The requested API version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ApplicationData"
+            }
+          },
+          "404": {
```

Comment:
Status codes listed under `responses` are considered success, so no error/exception will be propagated from the API call which I suspect isn't what you want.  If you want to keep this here for documentation purposes and have it be considered an error you must apply the [x-ms-error-response](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-error-response) attribute.

Reply:
Hi,
We do not want to propagate exceptions for 404. We use it a way to query if something exists or not.

Reply:
This authoring doesn't achieve that.  The 404 will be considered a success and the API will return an empty ApplicationData.

Reply:
Fixed this.